doc/changelog/05-tactic-language/13914-ltac2-cast-fun-return.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  Allow the presence of type casts for return values in Ltac2
```

Comment:
Since you also allow type casts for lets, this description is not entirely correct. How about "Allow the presence of type casts for function return and let values in Ltac2" ?